securebanking-openbanking-uk-obie-datamodel/src/test/java/uk/org/openbanking/testsupport/payment/OBWriteFileConsentTestDataFactory.java

```diff
@@ -60,10 +69,17 @@ public static OBWriteFile2DataInitiation aValidOBWriteFile2DataInitiation() {
                 .supplementaryData(new OBSupplementaryData1());
     }
 
+    public static OBWriteFile2DataInitiation aValidOBWriteFile2DataInitiationMandatoryFields() {
+        return new OBWriteFile2DataInitiation()
+                .fileType("UK.OBIE.pain.001.001.08")
+                .fileHash("m5ah/h1UjLvJYMxqAoZmj9dKdjZnsGNm+yMkJp/KuqQ");
```

Comment:
Is this a real hash for a test file which we intend to upload or is it just an example hash?